openmrs/i18n/clinical/locale_fr.json

```diff
@@ -80,5 +80,7 @@
   "DASHBOARD_UPCOMING_APPOINTMENTS_KEY": "Upcoming Appointments",
   "DASHBOARD_UPCOMING_APPOINTMENTS_KEY_LINK": "List View",
   "DASHBOARD_NO_PAST_APPOINTMENTS_KEY": "No past appointments",
-  "DASHBOARD_NO_UPCOMING_APPOINTMENTS_KEY": "No upcoming appointments"
+  "DASHBOARD_NO_UPCOMING_APPOINTMENTS_KEY": "No upcoming appointments",
```

Comment:
Aucun rendez-vous à venir is the translation in french

Reply:
We did not add/modify the No upcoming appointments translation. the ',' at the end is the only addition to this line

Reply:
I agree...My point was, can we change this to a proper french text? So that this wouldn't be taken up as a separate task in internationalization

Reply:
In the rest of the locale files, the translation is already there

Reply:
I will tag is as out of scope for this PR and may be create a new JIRA card for all the incorrect localizations.